ui/src/components/DeployConfirm.tsx

```diff
@@ -50,7 +50,7 @@ export default function DeployConfirm(props: DeployConfirmProps): JSX.Element {
             <Form.Item
                 {...layout}
                 label="Status"
-                style={(props.deployment.statuses && props.deployment.statuses.length > 0)? {marginBottom: 0} : {}}
+                style={(props.deployment.statuses && props.deployment.statuses.length > 0)? {marginBottom: 0} : {marginBottom: 12}}
```

Comment:
It makes a look normal.

![image](https://user-images.githubusercontent.com/17633736/140746260-f04088b3-69fd-43eb-9a20-4972ed047d36.png)